### PR TITLE
Feat: #110 Manage hd wallets

### PR DESCRIPTION
**Pull request by hoailinh1210**
### Summary (Please recap what you changed or fixed)
- Manage HD wallets
- Allow user to create a new wallet account

### Checklist (If you won't pass this checklist please comment why)
image

-   [x] I removed all comments or unnecessary code.
-   [x] Provide the screenshots due to UI changes or bug fixed
-   [x] My code has covered these test cases (please list down your tested cases):

- User can create a new wallet account with a default account
- User can select wallet account
- User can transfer from the selected account
- Keep the selected account when the user opens the extension again
Handle another task:
- Require password if service worker not working or User info is not found
- Restore account based on "Casper" token

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
 		"bootstrap-icons": "^1.5.0",
 		"browserslist": "^4.13.0",
 		"casper-js-sdk": "^2.9.1",
-		"casper-storage": "^0.2.4",
+		"casper-storage": "^0.2.5",
+		"clsx": "^1.2.1",
```
[the remark this answers is not in the document]
Comment (hoailinh1210) [reply client/package.json:26]: Hi @duc-q, `clsx` package in react-toastify, we should add `clsx` to our packages to manage it and avoid depending on another package.

**Comment by duc-q** [comment]
@hoailinh1210  

Please help pull latest changes to your branch, I've updated some changes related to styling, mostly removing `!important` from CSS and improve a bit on UX. 
Previously User doesn't acknowledge that he might have more than 5 wallets. On Chrome it won't show vertical scrollbar.
Improved: We show just a bit of extra info from the 6th one, enough to let user know he can scroll down for more.

image

Cheers

**Comment by gaupoit** [comment]
hi @hoailinh1210 , is it ready for review and testing? Have you resolved all feedbacks last week we mentioned?

**Comment by hoailinh1210** [comment]
Yes @gaupoit, I resolved all comment

**Comment by hoailinh1210** [comment]
> @hoailinh1210
> 
> Please help pull latest changes to your branch, I've updated some changes related to styling, mostly removing `!important` from CSS and improve a bit on UX. Previously User doesn't acknowledge that he might have more than 5 wallets. On Chrome it won't show vertical scrollbar. Improved: We show just a bit of extra info from the 6th one, enough to let user know he can scroll down for more.
> 
> image
> 
> Cheers

Hi @duc-q, I checked it. It looks good to me,
Thanks 👯 

> hi @hoailinh1210, is it ready for review and testing? Have you resolved all feedback last week we mentioned?

It is ready for review, and all feedback is resolved. I just create another PR for displaying the balance of wallets too.

**Comment by gaupoit** [comment]
Screen Shot 2022-09-26 at 15 59 57

hi @hoailinh1210 , seem to miss the edit icon on the right of "account name". (Account 1)

**Comment by hoailinh1210** [comment]
> Screen Shot 2022-09-26 at 15 59 57
> 
> hi @hoailinh1210 , seem to miss the edit icon on the right of "account name". (Account 1)

Hi @gaupoit, I just updated it

image

**Review by djhahe — APPROVED** [review]

**Pull request merged**